src/features/surveys/features/SurveyDisplay/components/OneQuestionView/OneQuestionView.tsx

```diff
@@ -3,11 +3,14 @@ import Header from 'shared/components/Header/Header';
 import Progressbar from 'shared/components/ProgressBar/ProgressBar';
 import { AnswersComponentFactory } from 'features/surveys/features/SurveyDisplay/components/AnswersComponent/AnswersComponentFactory';
 import { useSurveyDisplayContext } from 'features/surveys/features/SurveyDisplay/context';
+import { useSurveyCreatorContext } from 'features/surveys/features/SurveyCreator/managers/createSurveyManager/context';
 
 export default function OneQuestionView() {
   const { isAnswering, formData, activeQuestionIndex } =
     useSurveyDisplayContext();
 
+  const { surveyOptions } = useSurveyCreatorContext();
```

Comment:
Still we should not use `useSurveyCreatorContext` here. We should take `formData.hideProgressBar` from `useSurveyDisplayContext`

Reply:
Otherwise that option will not work while sharing survey